packages/core/src/pill-input/PillInput.css

```diff
@@ -273,6 +273,7 @@
   background: var(--salt-content-foreground-highlight);
   color: var(--salt-content-primary-foreground);
   --saltIcon-color: var(--salt-content-primary-foreground);
+  border-color: transparent;
```

Comment:
Design spec needs to be updated for this

Reply:
cc @yunjungyeh can't remember what you used here.

I raised this with @pseys last time [we looked at this](https://github.com/jpmorganchase/salt-ds/pull/3925) and even though this should probably use a token this was the easiest solution for now.